src/index.html

```diff
@@ -11,7 +11,174 @@
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="input-group" for="fullDateOfBirthday">
+          Full date of birthday:
+          <input
+            type="date"
+            name="fullDateOfBirthday"
+          >
+        </label>
+
+        <label class="input-group" for="term">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset-group">
+        <legend>
+          Registration
+        </legend>
+
+        <label class="input-group">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="input-group">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="4"
+            maxlength="9"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset-group">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <div class="input-group">
+
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              name="loveCats"
+              value="Y"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="N"
+            >
+            No
+          </label>
+
+        </div>
+
+        <label class="input-group">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            value="#000"
+          >
+        </label>
+
+        <label class="input-group">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="timeToGoToBed"
+          >
+        </label>
+
+        <label class="input-group">
+          What are you favorite brand of color?
+          <select name="favoriteBrandOfColor" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="input-group">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rateOurWork"
+            min="1"
+            max="10"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset-group">
+        <legend>
+          Additional info:
+        </legend>
+
+        <label class="input-group">
+          Comments:
+          <textarea
+            name="comments"
+            minlength="50"
+            maxlength="200"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
<img width="308" alt="image" src="https://user-images.githubusercontent.com/53611910/208319329-aa5e306e-5529-42e0-8d8b-05de1cc0c547.png">

Reply:
Hi, @ivanchuk-yurii 
 thanks for your comment, but when I fixed like the suggested in comment, I had a lint issue 
`159:11  error  Tag’s closing bracket should    (closing-and-opening-on-seme-
                be on the same level as         level)
                opening one`

Please, could you give me several tips? How to solve it?

Reply:
also, I try to find and solve this issue according to this description  https://www.npmjs.com/package/@mate-academy/html-lint . But it didn`t help

Reply:
What if you try like this

```
          <textarea 
            name="comments" 
            minlength="50"
            maxlength="200"
         ></textarea>
```

Reply:
![image](https://user-images.githubusercontent.com/17496682/209332177-f6557646-f4af-4c71-a432-7d81bfe5fa77.png)

Reply:
@Viktor-Kost it didn`t help

Reply:
![image](https://user-images.githubusercontent.com/17496682/209338093-d8eaef7e-bc9b-4312-a983-1645a26dbf8d.png)


is this the right option?